tests/integration/examples_test.py

```diff
@@ -70,3 +71,23 @@ def test_dns_edit(eth1_up):
         DNS.SERVER: [],
         DNS.SEARCH: [],
     }
+
+
+def test_add_remove_routes(eth1_up):
+    """
+    Test adding, downing and removing a vlan
+    """
+    route = {
+        Route.DESTINATION: '198.51.100.0/24',
+        Route.METRIC: '150',
+        Route.NEXT_HOP_ADDRESS: '192.0.2.1',
+        Route.NEXT_HOP_INTERFACE: 'eth1',
+        Route.TABLE_ID: '254'
+    }
+    with example_state('eth1_add_route.yml',
+                       cleanup='eth1_del_all_routes.yml') as desired_state:
+        assertlib.assert_state(desired_state)
+        with example_state('eth1_del_all_routes.yml') as desired_state:
```

Comment:
Why do you need these two lines? Upon the previous contextmanager exit, the `cleanup` is called which does the exact thing.

Reply:
Removed. Thanks.

Reply:
BTW. Do you why we do it for `test_add_down_remove_vlan`:

```python
    with example_state(
        'vlan101_eth1_up.yml', cleanup='vlan101_eth1_absent.yml'
    ) as desired_state:
        assertlib.assert_state(desired_state)
        with example_state('vlan101_eth1_down.yml') as desired_state:
            assertlib.assert_absent(vlan_ifname)

    assertlib.assert_absent(vlan_ifname)
```

Reply:
One is `down` and the other is `absent`.